sympy/combinatorics/perm_groups.py

```diff
@@ -3694,7 +3694,7 @@ def schreier_sims_random(self, base=None, gens=None, consec_succ=10,
         >>> from sympy.combinatorics.named_groups import SymmetricGroup
         >>> S = SymmetricGroup(5)
         >>> base, strong_gens = S.schreier_sims_random(consec_succ=5)
-        >>> _verify_bsgs(S, base, strong_gens) #doctest: +SKIP
```

Comment:
This is causing trouble randomly.

Reply:
@eagleoflqj The doctest should be skipped here?
Also, I don't know how things work after a PR gets merged like, this PR is merged now so if I want to make any further changes here then how should I be doing that?
The regular way we do changes to PR that is not merged yet,
OR
The PR needs to be unmerged first,
OR
A new PR should be created to make further changes?

Reply:
According to docstring of `schreier_sims_random`, this indeed can give wrong answer in a low probability. The easiest way is to add `# doctest: +SKIP` back in a new PR, though there might also be other way like use a fixed random seed.